products/jse/features/jse_metatag.feature

```diff
@@ -0,0 +1,78 @@
+Feature: Ensure Site Owners can create, edit and delete metatags. Ensure editors can create, edit and delete metatags
```

Comment:
Oh my. Looks there's some confusion here. Turns out that the site owner and editor can edit meta-tags on a per page basis. See the *Meta tags* in the vertical tabs at the bottom of the page:
https://sites.stanford.edu/jse-dev/node/158/edit

I don't know if we want either the site owner or the editor to access this page. I'll check.